INSTALL.md

```diff
@@ -15,50 +32,51 @@ Update WSL's software before continuing. If you chose Debian, Ubuntu, or another
 
-Otherwise, continue reading below for [the older Windows instructions](#windows).
+Otherwise, continue reading below for [the older Windows instructions](#windows-alternate).
+
 
+## Windows (alternate)
 
-## Windows
+*These instructions are the only option for older versions of Windows, but they also work for Windows 10 or 11 if you prefer not to use the [above](#windows-10-or-11).*
 
-Download [**Cygwin**](http://cygwin.com/install.html): **setup-x86_64.exe** for 64-bit Windows, **setup-x86.exe** for 32-bit.
+Download [**Cygwin**](http://cygwin.com/install.html): use the installer **setup-x86_64.exe** for 64-bit Windows, **setup-x86.exe** for 32-bit.
 
 Run setup and leave the default settings. At the "**Select Packages**" step, choose to install the following, all of which are in the "**Devel**" category:
 
 - `make`
 - `git`
 - `gcc-core`
 
-Double click on the text that says "**Skip**" next to each package to select the most recent version to install.
+Double click on the text that says "**Skip**" next to each package to select the most recent version to install. Click "**Next**" and go through the remaining installer screens until it's finished.
 
-Then follow the [**rgbds** install instructions](https://rgbds.gbdev.io/install#pre-built) for Windows with Cygwin to install **rgbds 0.5.2**.
+Once you have finished installing Cygwin, follow the [**rgbds** install instructions](https://rgbds.gbdev.io/install#pre-built) (click **Windows**, then **Cygwin, MSYS2**) to install **rgbds 0.5.2**.
 
-**Note:** If you already have an older rgbds, you will need to update to 0.5.2. Ignore this if you have never installed rgbds before. If a version newer than 0.5.2 does not work, try downloading 0.5.2.
+***Note** (ignore this if you have never installed **rgbds** before):  
+If you already have an older version of **rgbds**, you will need to update to **0.5.2**. If a version newer than **0.5.2** does not work, try downloading **0.5.2.***
 
-Now open the **Cygwin terminal** and enter the following commands.
+Now open the **Cygwin terminal** and enter the following commands. You can open the terminal by double-clicking `Cygwin.bat`, which is in ``C:\cygwin64\``.
 
-Cygwin has its own file system that's within Windows, at **C:\cygwin64\home\\*\<user>***. If you don't want to store pokered there, you'll have to change the **current working directory** every time you open Cygwin.
-
-For example, if you want to store pokered in **C:\Users\\*\<user>*\Desktop**:
+Cygwin has its own file system that's stored within Windows. The Cygwin terminal uses its own syntax, different from Windows, to refer to directories and objects inside the Cygwin file system. In Windows terms, the Cygwin terminal's default **current working directory** is `C:\cygwin64\home\<user>`; in Cygwin terms this same directory is called `/cygdrive/c/cygwin64/home/<user>`. (Replace `<user>` in the example path with your username).
```

Comment:
That middle sentence is a bit of a mouth full. How about?
```suggestion
Cygwin has its own file system that's stored within Windows. The Cygwin terminal uses a Unix-like file system. In Windows, the Cygwin terminal's default **current working directory** is `C:\cygwin64\home\<user>`; in Cygwin, this same directory is called `/cygdrive/c/cygwin64/home/<user>`. (Replace `<user>` in the example path with your username).
```

Reply:
How about this:
```suggestion
Cygwin has its own Unix-like file system that's stored within Windows but uses different syntax.
* In Windows, the Cygwin terminal's default **current working directory** is `C:\cygwin64\home\<user>`.
* In Cygwin, this same directory is called `/cygdrive/c/cygwin64/home/<user>`. 
(Replace `<user>` in the example path with your username.)
```